tests/fixtures/configs/test-model/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "model_type": "TestModel"
+}
```

Comment:
similar to [this](https://github.com/ArneBinder/pie-core/pull/76#discussion_r2197827443), can we move to `tests/fixtures/pretrained/model/`? (same for the bin file, of course)